watercore-api/src/main/java/me/lucyy/watercore/api/data/MySqlKey.java

```diff
@@ -0,0 +1,31 @@
+package me.lucyy.watercore.api.data;
+
+import java.io.Serializable;
+import java.util.Objects;
+import me.lucyy.watercore.api.data.DataKey;
+import me.lucyy.watercore.api.module.WaterModule;
+
+public class MySqlKey<T extends Serializable> extends DataKey {
```

Comment:
This class seems redundant and the overrides don't do anything minus the toString() which is used elsewhere in code